### PR TITLE
Update docker-compose file

### PR DESCRIPTION
**Pull request by vvaltia**
https://github.com/CSCfi/rems/issues/2210

Signed-off-by: Valtteri Valtia <valtteri.valtia@csc.fi>

# Definition of Done / Review checklist

## Reviewability
- [ ] link to issue
- [ ] note if PR is on top of other PR
- [ ] note if related change in rems-deploy repo
- [ ] consider adding screenshots for ease of review

## API
- [ ] API is documented and shows up in Swagger UI
- [ ] API is backwards compatible or completely new
- [ ] Events are backwards compatible

## Documentation
- [ ] update changelog if necessary
- [ ] add or update docstrings for namespaces and functions
- [ ] components are added to guide page
- [ ] documentation _at least_ for config options (i.e. docs folder)
- [ ] ADR for major architectural decisions or experiments

## Different installations
- [ ] new configuration options added to rems-deploy repository
- [ ] instance specific translations (i.e. LBR kielivara)

## Testing
- [ ] complex logic is unit tested
- [ ] valuable features are integration / browser / acceptance tested automatically

## Accessibility
- [ ] all icons have the aria-label attribute
- [ ] all fields have a label
- [ ] errors are linked to fields with aria-describedby
- [ ] contrast is checked
- [ ] conscious decision about where to move focus after an action

## Follow-up
- [ ] new tasks are created for pending or remaining tasks
- [ ] no critical TODOs left to implement

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -44,8 +44,6 @@ Point your browser to <http://localhost:3000>
 
 Build the image and initialize the database:
 
-    lein uberjar
-    docker-compose build
```
Comment (opqdonut) [inline docs/development.md:48]: I think the _development_ instructions should show how to run your current version of rems. Should we have a different docker-compose file for demoing REMS using the public cscfi/rems image?

Comment (opqdonut) [reply docs/development.md:48]: Alternatively, I guess we could just take this part out of docs/development.md and move it to some other place like docs/installing-and-upgrading.md. Nobody's using docker-compose.yml for development currently as far as I know.

**Review by opqdonut — CHANGES_REQUESTED** [review]

File: docs/development.md (hunk 1 above)
Comment (vvaltia) [reply docs/development.md:48]: Added second docker-compose file and moved the docker instructions to install doc

File: docs/installing-upgrading.md [hunk 2]
```diff
@@ -41,3 +41,39 @@ Some pointers for common tasks:
...
+    lein uberjar
+    docker-compose -f docker-compose-build.yml build
+    docker-compose -f docker-compose-build.yml up -d db
+    docker-compose -f docker-compose-build.yml run --rm -e COMMANDS="migrate test-data" app
+    docker-compose -f docker-compose-build.yml up -d app
+
+
+### Access rems
+
+    Point your browser to <http://localhost:3000>
```
Comment (opqdonut) [inline docs/installing-upgrading.md:70]: perhaps remove the indentation here as this isn't a shell snippet?

File: docs/installing-upgrading.md [hunk 3]
```diff
@@ -41,3 +41,39 @@ Some pointers for common tasks:
         java -Drems.config=path/to/your/rems/config -jar rems.jar migrate
 
 1. Restart rems. Remember to check the log for warnings, they might tell you about compatibility problems with the data or your configuration.
+
+# Running the application with Docker
```
Comment (opqdonut) [inline docs/installing-upgrading.md:45]: nice and simple :clap:

File: docs/installing-upgrading.md [hunk 4]
```diff
@@ -41,3 +41,39 @@ Some pointers for common tasks:
...
+
+### Option 1: Run rems from dockerhub
+
+    docker-compose up -d db
+    docker-compose run --rm -e COMMANDS="migrate test-data" app
+    docker-compose up -d app
+
+### Option 1.1: Use config file instead of environment variables
```
Comment (opqdonut) [inline docs/installing-upgrading.md:53]: could mention `simple-config.edn` here

File: docker-compose-build.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
+version: '2'
```
Comment (opqdonut) [inline docker-compose-build.yml:1]: these files could use a short comment at the top describing them

**Review by opqdonut — CHANGES_REQUESTED** [review]

File: docker-compose-config.yml [hunk 6]
```diff
@@ -1,3 +1,5 @@
+### Example file for running REMS using a config file with
```
Comment (opqdonut) [inline docker-compose-config.yml:1]: with what? :)

Comment (vvaltia) [reply docker-compose-config.yml]: Typo 8) Removed "with"

**Pull request merged**